lib/pentagon/Makefile

```diff
@@ -7,9 +7,9 @@ INFRASTRUCTURE_BUCKET = $(notdir ${INFRASTRUCTURE_REPO})
 BASENAME = $(notdir ${INFRASTRUCTURE_REPO})
 VPN_BUCKET = $(subst infrastructure,vpn,$(BASENAME))
 
-all: local_config_init install_requirements vpc vpc_id vpn working_cluster production_cluster
+all: local_config_init install_requirements vpc vpc_id nat_gateways configure_working_cluster configure_production_cluster #vpn
 
-destroy: destroy_vpn destroy_vpn_bucket destroy_vpc destroy_vpc_bucket
+destroy: destroy_vpn destroy_vpn_bucket destroy_working_cluster destroy_production_cluster destroy_vpc destroy_vpc_bucket 
```

Comment:
Please remove the trailing whitespace that is added to avoid a future change where this is removed again.